src/platform_impl/macos/window.rs

```diff
@@ -105,6 +106,20 @@ impl Window {
             Err(rwh_06::HandleError::Unavailable)
         }
     }
+
+    #[cfg(feature = "rwh_06")]
+    #[inline]
+    pub(crate) fn raw_display_handle_rwh_06(
+        &self,
+    ) -> Result<rwh_06::RawDisplayHandle, rwh_06::HandleError> {
+        struct UnsafeSendWrapper<T>(T);
+
+        unsafe impl<T> Send for UnsafeSendWrapper<T> {}
+
+        Ok(self
+            .maybe_wait_on_main(|w| UnsafeSendWrapper(w.raw_display_handle_rwh_06()))
+            .0)
+    }
```

Comment:
Like in #3288 this might have been incorrect to begin with.

Reply:
I mean, the display handle is empty on macOS, so it probably doesn't really matter?

Reply:
Ah, right, I can simplify this whole thing then.